skyportal/handlers/api/galaxy.py

```diff
@@ -121,12 +123,98 @@ def get(self, catalog_name=None):
         """
 
         catalog_name = self.get_query_argument("catalog_name", None)
+        localization_dateobs = self.get_query_argument("localizationDateobs", None)
+        localization_name = self.get_query_argument("localizationName", None)
+        localization_cumprob = self.get_query_argument("localizationCumprob", 0.95)
+        includeGeojson = self.get_query_argument("includeGeojson", False)
+
         query = Galaxy.query_records_accessible_by(self.current_user, mode="read")
         if catalog_name is not None:
             query = query.filter(Galaxy.catalog_name == catalog_name)
+
+        if localization_dateobs is not None:
```

Comment:
what should be returned if `localization_dateobs` is None? I dont see the logic for that.

Reply:
Currently it just returns the original query, which is the catalog (i.e. the current behavior of the endpoint).